controllers/contentlibrary/clustercontentlibraryitem/clustercontentlibraryitem_controller.go

```diff
@@ -111,9 +112,17 @@ func (r *Reconciler) ReconcileDelete(cclItem *imgregv1a1.ClusterContentLibraryIt
 		return nil
 	}
 
+	cvmi := &vmopv1.ClusterVirtualMachineImage{}
+	if err := r.Get(ctx, types.NamespacedName{Name: cvmiName}, cvmi); err != nil {
```

Comment:
Is there any preference between them? I am asking this because I noticed internally it has `type ObjectKey = types.NamespacedName` defined in the client library. Looks like we could have one less package (types) to import so I am going to update as suggested.